zebra-consensus/src/block.rs

```diff
@@ -100,6 +100,17 @@ impl VerifyBlockError {
             _ => false,
         }
     }
+
+    /// Returns a suggested misbehaviour score increment for a certain error.
+    pub fn misbehavior_score(&self) -> u32 {
+        // TODO: Adjust these values based on zcashd.
```

Comment:
```suggestion
        // TODO: Adjust these values based on zcashd. #XXXX
```

Should we create an issue for this ? I see the TODO is repeated a few times so it looks like something we might want or need to do at some point.

Reply:
I created issue #9258 for it and referenced it in the TODOs in https://github.com/ZcashFoundation/zebra/pull/9201/commits/88e0c9c0405ee533f6308394928a3f8de9f161d0.